aws/amazon-vpc-cni-k8s
# Use os.netLink.RuleAdd() APIs instead of using exec.cmd()

This PR  addresses issue #39 .
 * it removes calling exec.Cmd(""ip", "rule", "add", "not", "to", vpcCIDR.String()...)
 * instead, it uses os.netLInk.RuleAdd()/os.netLink.RuleDel() APIs 

**Tests Performed**
 * create a 1.9+ cluster using existing CNI 0.1.3
 * then perform rolling upgrade to CNI 0.1.4 
```
kubectl-osx apply -f  <cni.yaml which points to CNI 0.1.4>
``` 
* verify rolling upgrade finish OK 
```
kubectl-osx get ds -n kube-system
```
* deploy some pods and verify Pod-to-Pod ping OK
